src/components/CopyButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { StyleSheet, Clipboard } from 'react-native';
+
+import { typography } from 'app/styles';
+
+import { FlatButton } from './FlatButton';
+
+interface Props {
+  textToCopy: string;
+}
+
+export const CopyButton = ({ textToCopy }: Props) => {
+  const [isCopied, setIsCopied] = useState(false);
+  const copyToClipboard = async () => {
+    setIsCopied(true);
+    await Clipboard.setString(textToCopy);
```

Comment:
`await` seems to be redundant here

Reply:
I've checked a few articles to check if it's needed (for example: https://medium.com/the-react-native-log/react-native-basics-copy-to-clipboard-86023cda4175) but them seems to be outdated and now I've checked in the react native docs that `Clipboard` is deprecated -> https://reactnative.dev/docs/clipboard


So I'm gonna use `@react-native-community/clipboard` instead of `react-native` one and as I'm doing right now a ticket which also use that functionality - Wwuldn't be that a problem If I could just fix it in my current ticket?